src/main/java/com/philderbeast/prisonpicks/PrisonPicks.java

```diff
@@ -1,36 +1,25 @@
 package com.philderbeast.prisonpicks; 
 
-import com.sk89q.worldguard.bukkit.WorldGuardPlugin; 
+import java.io.File;
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.configuration.file.YamlConfiguration;
 import org.bukkit.plugin.java.JavaPlugin; 
 
 public class PrisonPicks extends JavaPlugin 
 {
 
     private static PrisonPicks instance; 
-
+    static File customYml;
+    static FileConfiguration customConfig;
+    
     @Override
     public void onLoad()
     {
+        
         Config.setConfigFolder(this.getDataFolder().getAbsolutePath()); 
         Config.reloadConfigs(); 
-
-        WorldGuardPlugin worldGuard = Util.getWorldGuard(); 
```

Comment:
why has the worldguard flag been removed?

Reply:
worldguard wont register custom flag, temp work around.

Reply:
nothing is more permant than a temporary fix, please do a proper fix rather then just remove it